include/loadfragment.h

```diff
@@ -0,0 +1,42 @@
+#ifndef LOADFRAGMENT_H
+#define LOADFRAGMENT_H
+
+#include "PR/ultratypes.h"
+#include "libc/stdint.h"
+#include "libc/stddef.h"
+
+extern s32 gOverlayLogSeverity;
```

Comment:
Just the smallest nit. I think we've been putting `externs` at the bottom of the file? I'm not sure if it's been consistent though so 🤷

Reply:
Yeah looks like we are pretty inconsistent. I would prefer leaving this here as is for now, and we can decide and clean them up later if that is okay.